examples/emoji_verification/src/main.rs

```diff
@@ -153,6 +153,26 @@ async fn login(homeserver_url: String, username: &str, password: &str) -> matrix
                                             .expect("Can't accept verification request");
                                     }
                                 }
+                                AnySyncMessageLikeEvent::KeyVerificationStart(
```

Comment:
Hmm why aren't we using event handlers in this example?

Reply:
I think the example predates the current event handler.

Reply:
Okay. I'll try to find some time to refactor it.